composer.json

```diff
@@ -22,7 +22,7 @@
         "php-http/discovery": "1.15.0",
         "symfony/dependency-injection": "5.4.16 || 6.0.16 || 6.1.8 || 6.2.0 || 6.2.1",
         "symfony/finder": "3.4.7 || 4.0.7 || 5.4.26 || 6.2.13 || 6.3.2",
-        "symfony/framework-bundle": "4.2.7 || 5.2.6 || 5.4.30",
+        "symfony/framework-bundle": "4.2.7 || 5.2.6 || >=5.4.30",
```

Comment:
This would now also target 6.*, no?

Reply:
True. Since I am at it, I'll test the same in Contao 5 and thus check whether we also need a conflict for Symfony 6.